hw/top_earlgrey/BUILD

```diff
@@ -429,7 +429,7 @@ silicon(
     name = "silicon_owner_sival_rom_ext",
     testonly = True,
     args = [
-        "--rcfile=",
+        "--rcfile=__builtin__/hyperdebug_teacup.json",
```

Comment:
I think this is an error.
`--rcfile=...` tells opentitantool which user-level config file to read, which contains commandline options like `--interface=foo` that the user would like set by default.

As far as I recall, `--rcfile` cannot parse a JSON file.

Reply:
I think you are right. I'll file a PR fixing it.